packages/reakit/src/Avatar/Avatar.js

```diff
@@ -4,9 +4,6 @@ import as from "../as";
 import Base from "../Base";
 
 const Avatar = styled(Base)`
-  border-radius: 50%;
-  width: 1.5em;
-  height: 1.5em;
```

Comment:
I'm not sure about `width` and `height`. Maybe we should create some theme variable for it.

Reply:
will do now